lms/templates/verify_student/make_payment_step_ab_testing.underscore

```diff
@@ -0,0 +1,108 @@
+<div id="wrapper-review" tab-index="0" class="wrapper-view make-payment-step">
+   <div class="review view">
+    <% if (!isActive ) { %>
+        <h2 class="page-title">
+            <%= _.sprintf(
+                gettext("Activate your account before upgrade to a  %(courseModeSlug)s certificate"),
+                { courseModeSlug:courseModeSlug }
+            ) %>
+        </h2>
+    <% } else if ( !upgrade ) { %>
+      <h2 class="page-title">
+        <%= _.sprintf(
+            gettext( "You are enrolling in: %(courseName)s"),
+            { courseName: '<span class="course-title">' + courseName + '</span>' }
+          ) %>
+      </h2>
+    <% } else { %>
+      <h2 class="page-title">
+        <%= _.sprintf(
+            gettext( "Upgrade to Verified Certificate: %(courseName)s"),
+            { courseName: '<span class="course-title">' + courseName + '</span>' }
+          ) %>
+      </h2>
+    <% } %>
+
+    <% if ( !isActive ) { %>
+        <p>
+            <%- gettext("Prior to completing this course purchase, you must activate your account.") %>
+            <%- gettext("Check your email for the activation link.") %>
+        </p>
+    <% } else { %>
+
+        <div class="certificate <%- courseModeSlug %>_icon">
+            <div class="purchase">
+              <p class="product-info"><span class="product-name"></span> <%- gettext( "Total" ) %>: <span class="price">$<%- minPrice %> USD</span></p>
+            </div>
+            <p>
+                <% if ( courseModeSlug === 'no-id-professional' || courseModeSlug === 'professional') { %>
+                    <%= _.sprintf(
+                        gettext( "Professional Certificate for %(courseName)s"),{  courseName: courseName }
+                     )%>
+                <% } else { %>
+                    <%= _.sprintf(
+                        gettext( "Verified Certificate for %(courseName)s"),{  courseName: courseName }
+                     )%>
+                <% } %>
+            </p>
+        </div>
+    <% } %>
+
+
+    <% if ( isActive ) { %>
+      <div class="payment-buttons is-ready center">
+        <input type="hidden" name="contribution" value="<%- minPrice %>" />
+        <input type="hidden" name="sku" value="<%- sku %>" />
+        <div class="pay-options">
+          <%
+          // payment buttons will go here
+          %>
+        </div>
+      </div>
+      <hr/>
```

Comment:
The horizontal rule layout attribute was removed in HTML5, please use CSS instead.